react/src/components/atoms/forms/Form/index.js

```diff
@@ -1,18 +1,17 @@
-import React, { Component } from 'react';
+import React, { Component, useContext } from 'react';
 import PropTypes from 'prop-types';
+import is from 'is';
+import deepEqual from 'fast-deep-equal';
 import { FormContext } from '../Input/context';
 
-const Form = (props) => (
-  <FormContext.Consumer>
-    {
-        (formContext) => (
-          <form className="ma__form-page" action="#">
-            {props.children(formContext)}
-          </form>
-          )
-      }
-  </FormContext.Consumer>
-);
+const Form = (props) => {
+  const formContext = useContext(FormContext);
+  return(
+    <form className="ma__form-page" action="#">
```

Comment:
just curious - i know this was there before - should the action be a passable prop with a default of `#`?

Reply:
It should yes!